web/app/components/share/text-generation/result/index.tsx

```diff
@@ -186,7 +187,7 @@ const Result: FC<IResultProps> = ({
     let isEnd = false
     let isTimeout = false;
     (async () => {
-      await sleep(1000 * 60) // 1min timeout
+      await sleep(TEXT_GENERATION_TIMEOUT_MS) // 1min timeout
```

Comment:
Please remove the confusing comment: 1min timeout.

Reply:
Thanks @iamjoel for reviewing the PR! Removed the comment.